Core/HLE/sceGe.cpp

```diff
@@ -408,6 +408,7 @@ int sceGeListSync(u32 displayListID, u32 mode) {
 
 static u32 sceGeDrawSync(u32 mode) {
 	//wait/check entire drawing state
+	hleEatCycles(500000); //HACK(?) : Potential fix for Crash Tag Team Racing and a few Gundam games
```

Comment:
Don't like this, this seems like a dangerous change that will affect the timing of many, many games. Of course, eating a small amount of cycles here makes sense, but that big an amount would seem to indicate that the problem really is that we are processing display lists too fast, and the game is buggy and stupidly relies on that.

If we find no other sensible solution, I could accept this behind a compat flag that only turns it on for the affected games.